pkg/kubemark/hollow_kubelet.go

```diff
@@ -142,6 +142,7 @@ func GetHollowKubeletConfig(opt *HollowKubletOptions) (*options.KubeletFlags, *k
 	c.FileCheckFrequency.Duration = 20 * time.Second
 	c.HTTPCheckFrequency.Duration = 20 * time.Second
 	c.NodeStatusUpdateFrequency.Duration = 10 * time.Second
+	c.NodeStatusReportFrequency.Duration = 5 * time.Minute
```

Comment:
thanks @sonyafenge for picking this up.

@Sindica this is to add back the line wrongly removed in https://github.com/CentaurusInfra/arktos/pull/955/files#diff-4ff678479fda4832300707ddef26710d07f76a2415edfc6504045e1c78a8f0bcL144

Reply:
no further comments on this. resolved. (no resolve button here)

Reply:
Help Peng to resolve the comment.